main.py

```diff
@@ -481,6 +481,100 @@ def delete_interface_metadata(self, request: Request) -> JSONResponse:
         self.notify_metadata_changes(interface, 'removed')
         return JSONResponse("Operation successful")
 
+    @staticmethod
+    def _get_tag_type(tag_type):
+        if tag_type not in {"1"}:
+            detail = f"The TAG type {tag_type} is not allowed."
+            raise HTTPException(400, detail=detail)
+        return tag_type
+
+    def _get_tag_ranges(self, request: Request):
+        """Get tag_ranges and check validity:
+        - It should be ordered
+        - Not unnecessary partition (eg. [[10,20],[20,30]])
+        - Singular intergers are changed to ranges (eg. [10] to [[10, 10]])
+        The ranges are understood as [inclusive, inclusive]"""
+        content_type_json_or_415(request)
+        tag_ranges = get_json_or_400(request, self.controller.loop)
+
+        for key, ranges in tag_ranges.items():
+            self._get_tag_type(key)
+            last_int = None
+            for i in range(0, len(ranges)):
+                if type(ranges[i]) is int:
```

Comment:
It's great to also have either `int` and `[int]` len 1, as you've done here, but let's also capture this as union type in the spec, all supported formats should be clear whoever is reading the spec and when the front-end will be implemented too. Speaking of front-end, if you could map an issue in the backlog for front-end, if you could also confirm if network engineers would be OK without any front-end to configure the tag ranges too that'd be great, I think we haven't discussed front-end in the issue discussion.

Reply:
On `openapi` added new component `Tag_ranges` specifying `anyOf` for array and integer with example `[[1, 500], 2096, [3001]]`

Reply:
I have not confirmed the front-end configuration yet.

Reply:
Talked about it and opened an [issue](https://github.com/kytos-ng/kytos/issues/411)

Reply:
Cool. Let's close this thread then. Appreciated you mapping and confirming the UI issue too.